docs/labs.rst

```diff
@@ -0,0 +1,668 @@
+===============================
+Labs - Experimental APIs
+===============================
+
+
+The pi-top Python SDK Labs are a set of classes which are being provided as
+experiments in exciting new ways to interact with your device. Everything in
+Labs is subject to change so use at your own risk.
+
+Web
+====
+
+The Web Labs have been created with the goal of giving users the power to create
+websites that run on the pi-top, and so can control the pi-top remotely. To
+achieve this we have added a number of classes which each serve a different
+purpose:
+
+- WebServer for simple websites
+- WebController for building websites that interact with your pi-top
```

Comment:
```suggestion
- WebController: 'batteries included' for building websites that interact with your pi-top
```

Reply:
how about `WebController: 'batteries included' WebServer for building websites that interact with your pi-top`?